src/recon_buildblock/ProjMatrixByBin.cxx

```diff
@@ -171,19 +171,23 @@ set_up(
 
 /*!
     \warning Preconditions
-    <li>abs(axial_pos_num) fits in 17 bits</li>
-    <li>abs(tangential_pos_num) fits in 11 bits</li>   
+    <li>abs(axial_pos_num) fits in 13 (4095) bits
+    <li>abs(tangential_pos_num) fits in 10 (1024) bits
+    <li>abs(tof_pos_num) fits in 7 bits (127)
```

Comment:
not enough...

Reply:
Needs more, but how many more bits? Is this heuristic?

Reply:
heuristic (who knows how many TOF bins a future scanner will have?). So I've given it plenty now by going to 64 bit